### ISSUE TITLE
bug(material-select): Failing accessibility score due to aria roles

### ISSUE THREAD

**Issue by heintj** [angular/components#21354]
#### Reproduction

Steps to reproduce:
1. Navigate to https://material.angular.io/components/select/overview
2. Go to examples
3. Run a plugin which checks accessibility scores of a page. 
I ran axe and lighthouse (lighthouse I believe is always included in google chrome) and both are failing on aria roles.

#### Expected Behavior
Accessibility score of 100

#### Actual Behavior
Accessibility score of 91 (lighthouse which is included in google chrome)
<img width="1171" alt="Schermafbeelding 2020-12-15 om 11 16 01" src="https://user-images.githubusercontent.com/39337718/102202453-9be7b600-3ec7-11eb-81d2-8975c4ffed92.png">

The axe plugin is telling me as follows:
Fix the following:
Required ARIA child role not present: textbox.
For each of the mat-selects on the page. 

I'm seeing same results on my app, which is running with @angular/material 10.2.7 and latest of angular 10.x.x.

Currently the role of the mat-select is combobox, which according to documentation of a11y indeed requires a child role of textbox: https://www.digitala11y.com/combobox-role/

#### Environment

- Angular: latest of 10.x.x and whatever is running in the documentation
- CDK/Material: 10.2.7 and 11.0.3
- Browser(s): I am using google chrome latest

**Comment by macjohnny** [comment]
related to https://github.com/angular/components/pull/20082

**Comment by macjohnny** [comment]
The warning occurs because of the following unmet requirement

https://www.w3.org/TR/wai-aria-1.1/#combobox
```
Authors MUST ensure an element with role combobox contains or owns a text input element with role textbox or searchbox and that the text input has aria-multiline set to false.
```
Adding an input element with `role="textbox"` in the select template fixes the issue.

**Comment by crisbeto** [comment]
This is working as expected and accessibility checking tools haven't been updated to account for the new pattern. See the discussion after https://github.com/angular/components/pull/20082#issuecomment-686622547.

**Comment by heintj** [comment]
thanks! :)
